src/electron/index.ts

```diff
@@ -326,6 +315,7 @@ function main() {
       app.setLoginItemSettings({openAtLogin: true, args: [Options.AUTOSTART]});
     }
 
+    mainWindow = createWindow();
```

Comment:
tangential: should we minimize the window on auto-start?

Reply:
I don't know. I added a TODO and I want to update auto start in another PR to:
- setup auto start when the vpn is connected and tear down on disconnection. This will prevent unnecessary residue.
- clear tunnel store in case of error, so it doesn't keep crashing
- always connect if the tunnelStore is set at start. This will enable auto start on Linux